hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -2,17 +2,17 @@ import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  kotlin("jvm") version "1.9.20"
-  kotlin("plugin.spring") version "1.9.20"
+  kotlin("jvm") version "1.9.22"
+  kotlin("plugin.spring") version "1.9.22"
   id("maven-publish")
   id("signing")
   id("com.adarshr.test-logger") version "4.0.0"
-  id("com.github.ben-manes.versions") version "0.49.0"
+  id("com.github.ben-manes.versions") version "0.50.0"
   id("se.patrikerdes.use-latest-versions") version "0.2.18"
-  id("io.spring.dependency-management") version "1.1.3"
-  id("org.jlleitschuh.gradle.ktlint") version "11.6.1"
-  id("org.owasp.dependencycheck") version "8.4.2"
-  id("org.springframework.boot") version "3.1.5"
+  id("io.spring.dependency-management") version "1.1.4"
+  id("org.jlleitschuh.gradle.ktlint") version "12.0.3"
+  id("org.owasp.dependencycheck") version "9.0.8"
```

Comment:
I'd keep this at 8.x otherwise you'll have to mirror the NVD feed

Reply:
Thanks, will do.